src/client/classes/Streamer.js

```diff
@@ -0,0 +1,128 @@
+const {
+    createAudioPlayer,
+    createAudioResource,
+    AudioPlayerStatus,
+    NoSubscriberBehavior
+} = require("@discordjs/voice");
+
+module.exports = class {
+    constructor() {
+        this.map = new Map();
+        this.mode = null;
+        this.logger = require("../funcs/logger.js");
+    }
+
+    init(client){
+        if(!client.config.streamerMode) return;
+
+        switch(client.config.streamerMode){
```

Comment:
switch can be just replaced with `this.mode = client.config.streamerMode ?? "manual"`

Reply:
This can't be used because there needs to be verification that values are following required choices. If I would use suggested solution it will make it possible that this.mode value can be other than auto or manual.